google-cloud-bigquery/samples/snippets/query_legacy_sql.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START bigquery_query_legacy]
+require "google/cloud/bigquery"
+
+def query_legacy_sql
```

Comment:
Can we please keep the name of the file and method name as `query_legacy` so that it's consistent with the region tag?

Reply:
Done.